src/main/java/com/eternalcode/parcellockers/database/ParcelLockerDatabaseService.java

```diff
@@ -0,0 +1,183 @@
+package com.eternalcode.parcellockers.database;
+
+import com.eternalcode.parcellockers.ParcelCache;
+import com.eternalcode.parcellockers.exception.ParcelLockersException;
+import com.eternalcode.parcellockers.parcel.ParcelLocker;
+import com.eternalcode.parcellockers.shared.Position;
+import io.sentry.Sentry;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class ParcelLockerDatabaseService {
```

Comment:
Chowaj sobie to lepiej za interfejsem.

Reply:
Póki co raczej nie ma takiej potrzeby, jest tylko 1 implementacja tego, jeżeli będę chciał kiedyś sobie rozbudować ten system na więcej repo, to pewnie tak zrobię, temat podlega dyskusji oczywiście

Reply:
Nie pierwszy raz już zmieniasz implementację z tego co pamiętam i lepiej żeby kod biznesowy nie był zależy od takich zmian.

Reply:
tworzenie takiego interfejsu wiąże się też z https://github.com/EternalCodeTeam/ParcelLockers/pull/47#discussion_r1193133311
+ wtedy biznes wytycza jak ma się zachowywać implementacja. a nie odwrotnie